app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1204,6 +1228,107 @@ impl Deref for GraphEditorModelWithNetwork {
     }
 }
 
+
+impl GraphEditorModelWithNetwork {
+    pub fn new(app: &Application, cursor: cursor::Cursor, frp: &Frp) -> Self {
+        let network = frp.network.clone_ref(); // FIXME make weak
```

Comment:
fix left

Reply:
This is part of moved code, not written by me.

Reply:
It seems to be non-trivial to change, so I would leave the "fixme".